package.json

```diff
@@ -40,22 +40,27 @@
     "nodes"
   ],
   "dependencies": {
+    "body-parser": "^1.19.0",
+    "request": "^2.88.0",
+    "url": "^0.11.0",
```

Comment:
Do we use `node-url` as this dependency points to? Otherwise, we do not need this as `node` has a module called `url`.

Reply:
You're right.